### PR TITLE
Implement `rank_initial_prompts` for web

### LINKED ISSUE #32: Implement `rank_initial_prompts` for web
The `rank_initial_prompts` task type needs to display these fields:
*  `prompts`: An array of prompts

It takes the following interaction type:
*  `type`: `post_ranking`
*  `ranking`: An array of ints representing the preferred ordering of the `prompts`, indexed by zero.  example: `[2,1,0,3]` for a series of 4 prompts.

### PR DESCRIPTION
**Pull request by AbdBarho**
Closes #32

I wanted to implement drag & drop sorting but I it is too much work.

The results are handed to the api where the best prompt has index 0, and the worst is the last.

### CODE REVIEW

File: website/src/pages/evaluate/rank_initial_prompts.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import { ArrowUpIcon, ArrowDownIcon } from "@heroicons/react/20/solid";
+import Head from "next/head";
+import { useState } from "react";
+import useSWRImmutable from "swr/immutable";
+import useSWRMutation from "swr/mutation";
+import clsx from "clsx";
```
Comment (fozziethebeat) [inline website/src/pages/evaluate/rank_initial_prompts.tsx:6]: Could this be ordered above alphabetically?

File: website/src/pages/evaluate/rank_initial_prompts.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+import { Button } from "src/components/Button";
+
+const RankInitialPrompts = () => {
+  const [tasks, setTasks] = useState([]);
+  const [ranking, setRanking] = useState<number[]>([]);
+
+  const { isLoading } = useSWRImmutable("/api/new_task/rank_initial_prompts", fetcher, {
+    onSuccess: (data) => {
+      console.log(data);
```
Comment (fozziethebeat) [inline website/src/pages/evaluate/rank_initial_prompts.tsx:19]: Let's drop these console logs now (i'll clean them up elsewhere later)

File: website/src/pages/evaluate/rank_initial_prompts.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import clsx from "clsx";
+
+import fetcher from "src/lib/fetcher";
+import poster from "src/lib/poster";
+
+import { Button } from "src/components/Button";
+
+const RankInitialPrompts = () => {
+  const [tasks, setTasks] = useState([]);
+  const [ranking, setRanking] = useState<number[]>([]);
```
Comment (fozziethebeat) [inline website/src/pages/evaluate/rank_initial_prompts.tsx:15]: Can you add the structure of this as a comment (basically what you wrote in the PR description)?  I think that'll help others that wander into looking at this.

**Review by fozziethebeat — APPROVED** [review]

File: website/src/pages/evaluate/rank_initial_prompts.tsx (hunk 1 above)
Comment (AbdBarho) [reply website/src/pages/evaluate/rank_initial_prompts.tsx:6]: gladly.

might be worth adding an eslint rule:

https://eslint.org/docs/latest/rules/sort-imports

should I create a new issue for this?

Comment (fozziethebeat) [reply website/src/pages/evaluate/rank_initial_prompts.tsx]: yes please~

**Pull request merged**